.make/test.mk

```diff
@@ -124,6 +124,12 @@ ALL_PKGS_EXCLUDE_PATTERN = "vendor\|account\/tenant\|app\'\|tool\/cli\|design\|c
 GOANALYSIS_PKGS_EXCLUDE_PATTERN="vendor|account/tenant|app|client|tool/cli"
 GOANALYSIS_DIRS=$(shell go list -f {{.Dir}} ./... | grep -v -E $(GOANALYSIS_PKGS_EXCLUDE_PATTERN))
 
+# temporary directory for fabric8-test
+FABRIC8_TEST_DIR = $(CUR_DIR)/fabric8-test
```

Comment:
Please change to `FABRIC8_E2E_TEST_DIR = $(TMP_PATH)/fabric8-test`.

Then also add this below:

```Makefile
CLEAN_TARGETS += clean-e2e
.PHONY: clean-e2e
## Removes the end to end (e2e) test directories.
clean-e2e:
	-rm -rf $(FABRIC8_E2E_TEST_DIR)
```

Then a `make clean` will also wipe out the e2e test data.

Reply:
+1